restlicsharpclient/restliclient/transport/HttpRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using restlicsharpclient.restliclient.request;
+
+namespace restlicsharpclient.restliclient.transport
+{
+    public class HttpRequest
+    {
+        public ResourceMethod method { get; }
```

Comment:
At this level, there is no ResourceMethod concept, it only knows about HTTP method (GET, PUT, POST, DELETE, etc)